internal/util/testutil/testutil.go

```diff
@@ -20,6 +20,7 @@ import (
 	"testing"
 )
 
+// Ctx returns a new empty context.
```

Comment:
```suggestion
// Ctx returns test context.
```

In the future, it may return a context with a deadline/timeout for graceful cancelation, etc.